src/interactions/panZoomInteraction.ts

```diff
@@ -140,25 +133,44 @@ export class PanZoom extends Interaction {
    * By default, `centerValue` is the center of the x and y range of each scale.
    */
   public zoom(zoomAmount: number, centerValue?: Point) {
```

Comment:
nit: given this is a functional change, it might be wise to hedge against upsetting users with an optional third arg to disable the constraint.